lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -972,6 +993,78 @@ void _AddMissingTangents(mx::DocumentPtr& mtlxDoc)
     }
 }
 
+#if MX_COMBINED_VERSION >= 13900
```

Comment:
1.39 added a bitangent input on the normal map node. This allows right and left handed renderers to decide if the bitangent is cross(N, T) or cross(T, N). This means we must add the proper cross product node here to get normal maps to render.